Tools/Scripts/webkitpy/autoinstalled/twisted.py

```diff
@@ -24,5 +24,5 @@
 
 from webkitscmpy import AutoInstall, Package, Version
 
-AutoInstall.register(Package('twisted', Version(15, 5, 0), pypi_name='Twisted'))
+AutoInstall.register(Package('twisted', Version(21, 7, 0), pypi_name='Twisted'))
```

Comment:
Is this version still Python 2 compliant? How have we validated that bumping the version isn't going to break anything?

Reply:
I think we don't need to keep python2 compatibility. Also maybe we should align with `autoinstalled.buildbot.py` which uses `AutoInstall.register(Package('twisted', Version(20, 3, 0), pypi_name='Twisted'))`?

Reply:
Looks like 21.7.0 is incompatible with Python 2 (`Requires-Python: >=3.6.7` according to their file).

Could we use the old version if we're in Python 2? Or raise an exception if we hit this code-path in Python 2? I don't think this will be relevant in practice, I don't think we have any Python 2 code paths running `twisted`, but `webkitpy` still supports Python 2 in general.

Reply:
I decide to use `Version(20, 3, 0)` which is last python2 compatible version of twisted.